caf-audit-service-dropwizard/src/main/resources/config.yaml

```diff
@@ -19,3 +19,26 @@ server:
 
 logging:
   type: external
+
+health:
```

Comment:
As an alternative, we could also do this via code rather than update the config.yaml, but it results in a lot more changes:

https://github.com/CAFAudit/audit-service/pull/202/files

Reply:
This approach seems fine to me, especially if it is less work.